<launch>
	<env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>
	
	
	
	<node
		name="jog_teleop"
		pkg="s4a_teleop"
		type="jog_teleop.py"
		clear_params="true"
		output="screen"
		respawn="true"
	>
		<!-- Input topics -->
		<remap from="joy" to="joy" />
		<!-- Output topics -->
		<remap from="jog_pos_cmd" to="/s4a/jog_pos_ctrl/command" />
		<remap from="motors_freewheel" to="/s4a/motors_freewheel" />
	</node>

</launch>
